comeet/UI/Login/LoginViewController.swift

```diff
@@ -54,6 +54,9 @@ private extension LoginViewController {
     }
     
     func goToMenu(){
+        let userDefault = UserDefaults.standard
+        userDefault.set(false, forKey: "isRoomsGuideShown")
```

Comment:
Why do we want to persist UI information on user defaults? User defaults is mainly for user preferences. I think we should think of a different approach.